src/vs/workbench/contrib/terminal/browser/terminalTooltip.ts

```diff
@@ -109,7 +109,11 @@ export function refreshShellIntegrationInfoStatus(instance: ITerminalInstance) {
 	}
 	const combinedString = instance.capabilities.get(TerminalCapability.CommandDetection)?.promptInputModel.getCombinedString();
 	if (combinedString !== undefined) {
-		detailedAdditions.push(`Prompt input: \`${combinedString}\``);
+		const escapedPromptInput = combinedString
+			.replaceAll('<', '&lt;').replaceAll('>', '&gt;') 		 //Prevent escaping from wrapping
+			.replaceAll(/\((.+?)(\|?(?: (?:.+?)?)?)\)/g, '(<$1>$2)') //Escape links as clickable links
+			.replaceAll(/([\[\]\(\)\-\*\!\#\`])/g, '\\$1'); 		 //Comment most of the markdown elements to not render them inside
+		detailedAdditions.push(`Prompt input: <code>\n${escapedPromptInput}\n</code>`);
```

Comment:
isn't this just as simple as escaping inner backticks like this?

```ts
combinedString.replaceAll('`', '\\`')
```

Reply:
The current proposal seems far too complex for what this actually does imo, all it's doing is making debug information show a little nicer. Does this helper do the job?

https://github.com/microsoft/vscode/blob/1d4fe40748197f0478cdf6247d43ec133ccfc67b/src/vs/base/common/htmlContent.ts#L153-L156